pkg/controllers/disruption/helpers.go

```diff
@@ -79,7 +81,12 @@ func SimulateScheduling(ctx context.Context, kubeClient client.Client, cluster *
 		pods = append(pods, n.reschedulablePods...)
 	}
 	pods = append(pods, deletingNodePods...)
-	scheduler, err := provisioner.NewScheduler(log.IntoContext(ctx, operatorlogging.NopLogger), pods, stateNodes)
+	scheduler, err := provisioner.NewScheduler(
+		log.IntoContext(ctx, operatorlogging.NopLogger),
+		pods,
+		stateNodes,
+		lo.Ternary(options.FromContext(ctx).PreferencePolicy == options.PreferencePolicyIgnore, scheduling.IgnorePreferences, nil),
```

Comment:
Is this more easily understood in scheduling code as a bool?

Reply:
Makes sense -- I'll make the update

Reply:
I sort of like keeping the same parity with the options fields -- it also has nice parity with the current reserved capacity options for the scheduler.